app/uk/gov/hmrc/merchandiseinbaggagefrontend/model/declaration/Declaration.scala

```diff
@@ -119,7 +122,27 @@ object DeclarationJourney {
   val id = "sessionId"
 }
 
-case class Goods(typeOfGoods: String, countryOfPurchase: String, priceOfGoods: PriceOfGoods, taxDue: CurrencyAmount)
+case class Goods(typeOfGoods: String, countryOfPurchase: String, priceOfGoods: PriceOfGoods, taxDue: CurrencyAmount) {
+  def toSummaryList(implicit messages: Messages): SummaryList = {
+    val price =
+      s"${priceOfGoods.amount.value.formatted("%.2f")}, ${priceOfGoods.currency.name} (${priceOfGoods.currency.code})"
```

Comment:
the formatting here is due to change but that's a different ticket, this is fine for now.